pyproject.toml

```diff
@@ -22,7 +22,7 @@ dependencies = [
     "brainglobe-napari-io",
     "dask >= 2.15.0",
     "imio",
-    "imlib >= 0.0.26",
+    "brainglobe-utils",
```

Comment:
Does a minimum pin need to go in here?

Reply:
Nope, the lowest version of brainglobe-utils is essentially the version pinned here.